seisbench/models/aepicker.py

```diff
@@ -127,3 +127,27 @@ def classify_aggregate(self, annotations, argdict):
             )
 
         return sorted(picks)
+
+    def get_model_args(self):
+        model_args = super().get_model_args()
+        model_args = {
```

Comment:
This code will definitely work, but maybe it would look a bit cleaner without the dict comprehension. How about something like:
```python
for key in ["citation", "in_samples", "output_type", "default_args", "pred_sample", "labels", "sampling_rate"]:
    del model_args[key]
```
I think this would be more explicit

Reply:
Same for the other models.

Reply:
Sounds good, syntax changed to the above format for parsing any model keys.